### PR TITLE
[VS Code] changelog and readme

### PR DESCRIPTION
**Pull request by sunqinzheng**

### CODE REVIEW

File: contrib/pai_vscode/CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,17 @@ All notable changes to this project will be documented in this file.
...
+## [0.1.0] - 2019-01
 ### Added
 - Open job pages and dashboard pages in VS Code
 - Submit job to PAI cluster from VS Code
 - Open PAI's hdfs as a workspace folder
+
+## [0.2.0] - 2019-03
+### Added
+- Generate jsonc job config by default
+- Add an Open PAI view container (sidebar), includes
```
Comment (scarlett2018) [inline contrib/pai_vscode/CHANGELOG.md]: >  Add an Open PAI
OpenPAI

File: contrib/pai_vscode/README.md [hunk 2]
```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
 
 ![](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/simulate-job.gif)
 
+## Sidebar
```
Comment (scarlett2018) [inline contrib/pai_vscode/README.md]: 来个截图吧

File: contrib/pai_vscode/README.md [hunk 3]
```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
...
+HDFS Explorer and Job List view will be shown in the extension's view container (sidebar).
+
+### HDFS Explorer
+1. Connected to a PAI cluster's hdfs by double click the corresponding tree node.
```
Comment (scarlett2018) [inline contrib/pai_vscode/README.md]: 1. You are able to connect to an OpenPAI cluster's HDFS by double click its corresponding tree node.

File: contrib/pai_vscode/README.md [hunk 4]
```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
...
+2. The cluster's directory structure will be shown in the tree view. Right click the folder/file node and perform selected file system operations.
```
Comment (scarlett2018) [inline contrib/pai_vscode/README.md]: 2. The HDFS directory structure will be expended in the tree view. 
3. Right click the folder or file node to perform file system operations.

File: contrib/pai_vscode/README.md [hunk 5]
```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
...
+
+### Job List
+PAI cluster's latest jobs and jobs recently submitted by this extension will be shown in the job list tree view.
```
Comment (scarlett2018) [inline contrib/pai_vscode/README.md]: 1. Browse latest jobs in the OpenPAI cluster in job list tree view.
//question: how lastest? (in 1 day?)
2. Browse recent submitted jobs by local (extension) in job list tree view.
//加个图吧

File: contrib/pai_vscode/README.md [hunk 6]
```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
...
+
+- The job list will be auto refreshed every 10 seconds by default.
```
Comment (scarlett2018) [inline contrib/pai_vscode/README.md]: 3. Job list will auto refresh every 10 seconds.
//is the default value changable?

File: contrib/pai_vscode/README.md [hunk 7]
```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
...
+
+### HDFS Explorer
+1. Connected to a PAI cluster's hdfs by double click the corresponding tree node.
+2. The cluster's directory structure will be shown in the tree view. Right click the folder/file node and perform selected file system operations.
+
+### Job List
+PAI cluster's latest jobs and jobs recently submitted by this extension will be shown in the job list tree view.
+
+- The job list will be auto refreshed every 10 seconds by default. 
+- You can double click job's node to open PAI's job detail page in external browser.
```
Comment (scarlett2018) [inline contrib/pai_vscode/README.md]: 4. Double click job node to browse Job details in external browser.

File: contrib/pai_vscode/README.md (hunk 5 above)
Comment (sunqinzheng) [reply contrib/pai_vscode/README.md]: //10 jobs by default

**Pull request merged**